downloads/swan-lake-release-notes/swan-lake-alpha4/swan-lake-alpha4-release-note.md

```diff
@@ -276,32 +287,45 @@ handle secondWord = arrays:get(input, 1);
     }
 
 ```
+
 ##### Kafka Package Updates
```

Comment:
Updates to the `kafka` package or Updated the `kafka` package.

Reply:
Saying "Kafka Package Updates" should be fine because we have the main titles also as follows:

- Language Updates
- Runtime Updates
- Standard Library Updates
- Code to Cloud Updates
- Developer Tools Updates
- Ballerina Packages Updates
- Breaking Changes